engines/director/score.cpp

```diff
@@ -1560,26 +1723,28 @@ void Score::loadActions(Common::SeekableReadStreamEndian &stream) {
 			break;
 	}
 
-	bool *scriptRefs = (bool *)calloc(_actions.size() + 1, sizeof(bool));
-
-	// Now let's scan which scripts are actually referenced
-	for (uint i = 0; i < _frames.size(); i++) {
-		if ((uint)_frames[i]->_actionId.member <= _actions.size())
-			scriptRefs[_frames[i]->_actionId.member] = true;
-
-		for (uint16 j = 0; j <= _frames[i]->_numChannels; j++) {
-			if ((uint)_frames[i]->_sprites[j]->_scriptId.member <= _actions.size())
-				scriptRefs[_frames[i]->_sprites[j]->_scriptId.member] = true;
-		}
-	}
-
 	Common::HashMap<uint16, Common::String>::iterator j;
 
 	if (ConfMan.getBool("dump_scripts"))
```

Comment:
I wonder, does it mean that we will not dump whole movie scripts, only the played frames there? Or, perhaps, with the `_numFrames` precalculation, we could still dump all of them?